server/bleep/src/indexes/file.rs

```diff
@@ -213,50 +214,46 @@ impl Indexer<File> {
     /// If the regex filter fails to build, an empty list is returned.
     pub async fn fuzzy_path_match(
         &self,
-        repo_ref: &RepoRef,
+        project: Project,
         query_str: &str,
-        branch: Option<&str>,
         limit: usize,
```

Comment:
We need to keep the ability to filter these searches by branch. We could keep branch as an `Option<&str>`, and filter searches for all paths on that branch across the provided `RepoRefs`.

Reply:
We'd want to couple branches to `RepoRefs`, it doesn't make a lot of sense to, say, search across a list of repos in the `dev` branch, as they may not have one.